src/qt/forms/createwalletdialog.ui

```diff
@@ -62,64 +65,77 @@
+    <string>Private keys in the wallet will be encrypted with a passphrase of your choice. Addresses, transaction history and notes are not encrypted.</string>
    </property>
    <property name="text">
-    <string>Encrypt Wallet</string>
+    <string>Encrypt private keys</string>
    </property>
    <property name="checked">
-    <bool>true</bool>
+    <bool>false</bool>
+   </property>
+  </widget>
+  <widget class="QLabel" name="encrypt_wallet_label">
+   <property name="geometry">
+    <rect>
+     <x>20</x>
+     <y>90</y>
+     <width>130</width>
+     <height>21</height>
+    </rect>
+   </property>
+   <property name="text">
+    <string>Advanced options</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="disable_privkeys_checkbox">
    <property name="enabled">
-    <bool>false</bool>
+    <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
-     <y>80</y>
+     <y>115</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
-    <string>Disable private keys for this wallet. Wallets with private keys disabled will have no private keys and cannot have an HD seed or imported private keys. This is ideal for watch-only wallets.</string>
+    <string>Disable private keys for this wallet. Watch-only wallets have no private keys and cannot have an HD seed or imported private keys.</string>
    </property>
    <property name="text">
-    <string>Disable Private Keys</string>
+    <string>Watch-only</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="blank_wallet_checkbox">
    <property name="geometry">
     <rect>
      <x>20</x>
-     <y>110</y>
-     <width>171</width>
+     <y>135</y>
+     <width>220</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
-    <string>Make a blank wallet. Blank wallets do not initially have private keys or scripts. Private keys and addresses can be imported, or an HD seed can be set, at a later time.</string>
+    <string>Blank wallets do not initially have private keys or scripts. Private keys and addresses can be imported, or an HD seed can be set, at a later time.</string>
    </property>
    <property name="text">
-    <string>Make Blank Wallet</string>
+    <string>Manually import seed or keys</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="descriptor_checkbox">
    <property name="geometry">
     <rect>
      <x>20</x>
-     <y>140</y>
+     <y>155</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
-    <string>Use descriptors for scriptPubKey management</string>
+    <string>Use descriptors for scriptPubKey management.</string>
    </property>
    <property name="text">
-    <string>Descriptor Wallet</string>
+    <string>Use descriptors</string>
```

Comment:
"Descriptor wallet" seems better?

Reply:
The word "wallet" seems redundant. Also, you can mix and match, e.g. a watch-only descriptor wallet (not a watch-only wallet descriptor wallet)

Reply:
Hm, good points. WDYT about:

- Watch-only
- Blank
- Descriptors

or

- Watch-only
- Skip seed generation
- Use descriptors

Reply:
I guess we should mention what can be mixed with what.

Reply:
Watch-only implies blank (checkbox enforces that), other than that there's no restriction. See also the `createwallet` RPC documentation, which this is dumb mimic of.

The main point of this PR is to make v0.21 slightly more usable than it is now, because we no longer create a blank wallet. I'd rather not fine tune this too much, that's what #77 is for.